<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDSetupDlg</class>
 <widget class="QDialog" name="IDSetupDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>342</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>ID 设置</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2" colspan="3">
    <widget class="QGroupBox" name="FDBGroupBox">
     <property name="title">
      <string>来自因子库</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>因子表</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>因子</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;时点&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QDateTimeEdit" name="DateTimeEdit">
        <property name="currentSection">
         <enum>QDateTimeEdit::YearSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="IgnoreDTCheckBox">
        <property name="text">
         <string>忽略时点</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="FIDSelectTypeComboBox">
        <item>
         <property name="text">
          <string>覆盖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>并集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>交集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>左差右</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>右差左</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>对称差</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="SelectFDBIDButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="FTComboBox"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="FactorComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>ID 数</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="IDNumEdit">
     <property name="text">
      <string>0</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2" colspan="2">
    <widget class="QListWidget" name="IDListWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="IDGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="CodeLabel_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ID 类型&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>CodeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="IDTypeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="CodeLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;指数代码&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>CodeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="CodeEdit">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;截止日期&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QDateEdit" name="DateEdit">
        <property name="currentSection">
         <enum>QDateTimeEdit::YearSection</enum>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="CurrentCheckBox">
        <property name="text">
         <string>历史成分</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="SelectIDTypeComboBox">
        <item>
         <property name="text">
          <string>覆盖</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>并集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>交集</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>左差右</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>右差左</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>对称差</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="SelectIDButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>75</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="AcceptButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="RejectButton">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <widget class="QLineEdit" name="IDInputEdit">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CodeEdit</tabstop>
  <tabstop>SelectFDBIDButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
